src/server/user-controller.js

```diff
@@ -65,6 +65,7 @@ class UserController {
 
   handleGetLogoutPage(req, res) {
     req.logout();
+    res.clearCookie(this.serverConfig.sessionCookieName);
```

Comment:
We were not clearing the cookie on logout, so even logging out from prod was still making you unable to login from staging.

Reply:
This is by design, as a session doesn't require to be login, you can also have an anonymous session. But we don't use it, so it's fine to clear the cookie here. We just have to remember in the future, in case we store values in the server session.

Reply:
Oh, okay, thanks for the comeback.